src/SlashCommands.js

```diff
@@ -43,11 +43,27 @@ var commands = {
         return reject("Usage: /nick <display_name>");
     },
 
-    // Takes an #rrggbb colourcode and retints the UI (just for debugging)
+    // Changes the colorscheme of your current room
     tint: function(room_id, args) {
-        Tinter.tint(args);
-        return success();
-    },
+
+        if (args) {
+            var matches = args.match(/^(#([0-9a-fA-F]{3}|[0-9a-fA-F]{6}))( +(#([0-9a-fA-F]{3}|[0-9a-fA-F]{6})))?$/);
+            if (matches) {
+                Tinter.tint(matches[1], matches[4]);
+                var colorScheme = {}
+                colorScheme.primary_color = matches[1];
+                if (matches[4]) {
+                    colorScheme.secondary_color = matches[4];
+                }
+                return success(
+                    MatrixClientPeg.get().setRoomAccountData(
+                        room_id, "m.room.color_scheme", colorScheme
```

Comment:
no, it's different to the "it'd be nice if people's default avatars were consistent across clients", as that was effectively forcing default avatars on clients, which had to fight against the choice if they wanted to do something different.

This is a purely advisory hint of a colourscheme that the user explicitly chose for a room.  If clients support per-room colouring, then they have the option of picking from this matrix-wide datasource, or going off and implementing their own.  For instance, if we federated with Wire, which does a similar thing, we'd want to synchronise the colour-scheme with them.

That said, I guess I can make it vector specific for now, only to change it in future, but I consider it no more vector specific than (say) room tags.